packages/plugin-ext/src/plugin/env.ts

```diff
@@ -48,7 +56,32 @@ export class EnvExtImpl {
         this.queryParameters = queryParams;
     }
 
+    setLanguage(lang: string): void {
+        this.lang = lang;
+    }
+
     getClientOperatingSystem(): Promise<theia.OperatingSystem> {
         return this.proxy.$getClientOperatingSystem();
     }
+
+    get appName() {
+        return EnvExtImpl.APP_NAME;
+    }
+
+    get appRoot() {
+        return __dirname;
```

Comment:
Will this work in web worker?

Reply:
will check, I splitted class class for node-mac, I will see how to handle that
good catch

Reply:
@mmorhun : ok I've splitted in two classes, one for worker and one for nodejs

in worker env, we don't provide filesystem path as there is no FS BTW